content/learn/book/welcome/community/_index.md

```diff
@@ -5,9 +5,27 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: List places where the community congregates, and differentiate
+Bevy would not be what it is without the support from the community. The community can be found in many locations such as:
+- [Bevy's Github Page](https://github.com/bevyengine/bevy)
+- [Bevy's Twitter](https://twitter.com/bevyengine)
+- [Bevy's Discord](https://discord.gg/bevy)
 
-TODO: link to awesome-bevy and discuss crate ecosystem
+More can be found at this website's [community page](https://bevyengine.org/community).
 
-TODO: link to CONTRIBUTING.md and talk about how Bevy is maintained by Cart plus a broad community of volunteers.
-Link to Cart's donation page
+Our community is very active and welcoming.
+
+## Third-Party Crates
+
+Bevy has a lot of third-party crates associated with it, such as [bevy_retrograde](https://github.com/katharostech/bevy_retrograde), [bevy_spicy_networking](https://github.com/CabbitStudios/bevy_spicy_networking) and [heron](https://github.com/jcornaz/heron).
```

Comment:
We should be careful not to link to Katharos stuff without *very* clear license disclaimers. They seem like nice folks, but the license is a deal-breaker.